server/lib/sanity-check.js

```
@@ -4,6 +4,7 @@ const uuid = /[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}/;
 const empty = /^\s+$/;
 // const ftcontent = /\<\/?ft-content/;
 
+// COMPLEX: This *returns* true OR Error objects.  Be warned…
```

@i-like-robots this was super confusing…  my humble opinion that it would be more natural for this function to `throw` errors and consistently return a boolean rather than return both.